### ISSUE TITLE
[Bug]: Not being able to take your clothes off when holding "y" down in game 

### ISSUE THREAD

**Issue by TheKyle1** [CMGStudios/issue-tracker#190]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Im not sure if its just me but when im playing CMG, it's just stopped working so i cant hold down my "y" on my keyboard and take my clothes off one by one. it gives me the menu and the circle but doesnt let me click the clothes individually. 

### Expected behaviour

Cant take my clothes off when holding down "Y"

### Steps to reproduce

-

### Any Texture Packs? (If so list)

nope

### CMG User ID

789

**Comment by loganuk** [comment]
Do you have a clip of this happening so we can see?

**Comment by TheKyle1** [comment]
@loganuk Hello, yes ill get a clip for you. 

So as you can see in the clip, im opening the Y menu. and clicking on all the clothes and it doesnt come off and on

I have done everything from clearing cache and uninstalling fivem, still doesnt work. Was told to make a bug report.

Here is the video - https://youtu.be/M-Xp-B6J6h4

**Comment by TheKyle1** [comment]
@loganuk Any update on this?

**Comment by TheKyle1** [comment]
This issue is still ongoing, is there an update for this?

**Comment by crazychris2404** [comment]
Sorry for the late response,
Is this issue still occurring?

**Comment by TheKyle1** [comment]
@crazychris2404 Yes this issue is still ongoing. 

I have just given up but a fix would be amazing! 

I have also seen other people have this issue aswell. 

Thanks

**Comment by crazychris2404** [comment]
Do you happen to know if this is reproducible and does changing gender to male/female fix it ?

**Comment by crazychris2404** [comment]
@TheKyle1 Does using commands such as /shirt work? As it could be a issue with the GUI

**Comment by TheKyle1** [comment]
@crazychris2404 Yes using /shirt and /hat etc etc all work. That takes off and puts on my clothes. which is all I have been doing for the past couple months.

**Comment by crazychris2404** [comment]
Alright @arthur-rl this could be a issue affecting GUI for some players.

**Comment by tom-gun** [comment]
Closed, reported as being fixed
